tests/scalability/generate_brac_data/README.md

```diff
@@ -0,0 +1,46 @@
+# Brac Config Data Generator Script (first draft)
+
+## Introduction
+
+This script is to generate Brac config test data.
+
+Initially it only creates places, persons, users and reports (pregnancy, assessment and assessment follow up). Taking into account hierarchies.
+
+Changes where made in /test/factories/ and /test/scalability/
+Basically added new folders:
+/test/factories/brac and /test/scalability/generate_brac_data
+
```

Comment:
I think the readme shouldn't mention changes, rather it should describe what is currently present and how it is used.
```markdown
Relevant factories exist in `test/factories/brac`. 
```